storybooknative/src/Touchable.js

```diff
@@ -18,6 +18,7 @@ type Props = {|
   // Should the ripple render outside of the view bounds?
   +borderlessRipple?: boolean,
   +rippleColor?: string,
+  +width?: string | number,
```

Comment:
I don't really like mixed types like this. @tbergq can you think of a better way?

Reply:
Why not use flex instead?

Reply:
Do you mean alignSelf: "stretch"?

Reply:
I mean flex: 1, so it will take full width. If we want it to take any less, we can wrap it with another view, and it will by default take the full width of that view

Reply:
`flex: 1` should flex only in the flexDirection

Reply:
But I think our buttons have a fixed height, so it should respect that, then grow only in the width

Reply:
I tried your solution, but in emulator it actually stretches the button in its height and not width. 
I don't know what is happening or where I made an error.

Reply:
Yes, you are right, it was a bad advice, sorry 🙈